LSICORR/iocBoot/iocLSICORR-IOC-01/runIOC.bat

```diff
@@ -2,12 +2,23 @@
 setlocal
 set MYDIRBLOCK=%~dp0
 call C:\Instrument\Apps\EPICS\config_env_base.bat
-
+call dllPath.bat
 %HIDEWINDOW% h
 
 set EPICS_CAS_INTF_ADDR_LIST=127.0.0.1
 set EPICS_CAS_BEACON_ADDR_LIST=127.255.255.255
 
 set PYTHONUNBUFFERED=TRUE
 
-%PYTHON3W% %EPICS_KIT_ROOT%/support/lsicorr/master/LSi_Correlator.py --pv_prefix %MYPVPREFIX% --ioc_number 1
+set "GETMACROS=C:\Instrument\Apps\EPICS\support\icpconfig\master\bin\%EPICS_HOST_ARCH%\icpconfigGetMacros.exe"
+set "MYIOCNAME=LSICORR_01"
+echo PRE %REFL_MACROS%
```

Comment:
Just having `PRE` here is a bit odd, can you make it clear what is printing? Is the printing actually useful?